pylab/website/views.py

```diff
@@ -71,7 +71,26 @@ def about(request):
 
 def event_details(request, year, month, day, slug):
     event = get_object_or_404(Event, starts__year=year, starts__month=month, starts__day=day, slug=slug)
-    return render(request, 'website/event_details.html', {'event': event})
+    attendance = Attendance.objects.filter(event=event)
+    if request.method == 'POST':
+        form = website_forms.AttendanceForm(request.POST)
+        if form.is_valid():
+            instance = Attendance.objects.get(event=event, attendee=request.user)
+            instance.response = form.cleaned_data['response']
+            instance.save()
```

Comment:
These three lines should be replace by ```form.save()```. If you needed to execute more logic during form saving, you should override the ```AttendanceForm.save()``` method, but this is not the case.